agent/tool-scripts/base-tool

```diff
@@ -107,7 +107,7 @@ pidstat)
 	threads="false"
 	longoptions="${longoptions},options:,patterns:,threads"
 	tool_package_name="pbench-sysstat"
-	tool_package_ver="12.0.3"
+	tool_package_ver="12.0.3-1"
```

Comment:
As a "quick fix", this is fine; however, in general, it's annoying:  this is a weird place to enshrine this information (e.g., why isn't it in some sort of `requirements.txt` kind of place?), and you've got it in _two_ places -- here and at line 67 -- and I'm pretty sure that we don't want them to get out of sync (so, they should probably have a common definition which is referenced here).

Reply:
Agreed on all points.  This is something that we can address as we convert from `bash` to `Python3`, and by dropping the `sysstat` tools in favor of PCP.